R/helper_functions.R

```diff
@@ -0,0 +1,264 @@
+#' get_incidence_db_name: Get the right name for the data file to be saved
+#'
+#' @description
+#' Based on the user-specified parameters of cancer, race, sex, age, stage, and
+#' year, return an appropriate name for the database file to be saved
+#'
+#' @param cancer The type of cancer (character)
+#' @param race The race of the population (character)
+#' @param sex The sex of the population (character)
+#' @param age The age group of the population (character)
+#' @param stage The stage of the cancer (character)
+#' @param year The desired timespan for the data (character)
+#'
+#' @return
+#' The name of the database file to be saved to duckdb for this combination of
+#' parameters
+#'
+#' @examples
+#' get_incidence_db_name(
+#'    "all cancer sites",
+#'    "All Races (includes Hispanic)",
+#'    "both sexes",
+#'    "all ages",
+#'    "all stages",
+#'    "latest 5 year average"
+#' )
+#'
+
+get_incidence_db_name <- function(cancer, race, sex, age, stage, year){
```

Comment:
We should talk about this database schema, I'm not convinced this is the best way forward.